plugin/restore/mysql/mysql.go

```diff
@@ -123,6 +132,170 @@ func GetPITRDatabaseName(database string, timestamp int64) string {
 	return getSafeName(database, suffix)
 }
 
+// SyncBinlogs syncs the binlogs between the instance and `saveDir`, but exclude latest binlog
+func (r *Restore) SyncBinlogs(ctx context.Context, instance *api.Instance, saveDir string) error {
+	// 1. read the binlog file in saveDir
```

Comment:
IMO, we do not need trivial comments like this, because the code is simple enough to be self-explanatory.